components/infobox/wikis/callofduty/infobox_team_custom.lua

```diff
@@ -0,0 +1,46 @@
+---
+-- @Liquipedia
+-- wiki=callofduty
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local MatchTicker = require('Module:MatchTicker/Custom')
+local String = require('Module:StringUtils')
+
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local CustomTeam = Class.new()
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.getWikiCategories = CustomTeam.getWikiCategories
+	return team:createInfobox()
+end
+
+function CustomTeam:createBottomContent()
+	return MatchTicker.participant{team = self.pagename}
+end
+
+function CustomTeam:addToLpdb(lpdbData, args)
+	lpdbData.extradata.competesin = string.upper(args.league or '')
```

Comment:
you sure you want to store empty strings if this is unset?
alternative that doesn't store if unset:
```lua
	lpdbData.extradata.competesin = String.isNotEmpty(args.league) and args.leafue:upper() or nil
```